lib/puppet/provider/keystore/keytool.rb

```diff
@@ -1,51 +1,7 @@
+require 'puppet_x/certs/keystore'
```

Comment:
It may be nice to differentiate this somehow so that it's clear that this is just for the provider.

Reply:
Done. I introduced another directory layer.